js/anims-missiles.js

```diff
@@ -91,4 +100,23 @@ function listenKeypressMissiles() {
 
 createMissileEventHandler()
 listenKeypressMissiles()
+
+
+function incrementToll() {
+    var previousToll = parseInt(document.querySelector('#toll').innerHTML)
+    var currentToll = Math.round(previousToll + ((Math.random()+1)*1E6))
+    return currentToll
+}
+
 repeatAnims()
+
+function hitCityEffect(cityid) {
+    makeNoise('hitcity')
+    var explosion = document.querySelector('#city-' + cityid + ' .explosion')
+    clearTimeout(impacts[cityid - 1])
+    //we append a random query string to make the gif reliably start at zero when a new explosion occurs.
+    explosion.setAttribute('src', 'img/city-hit.gif' + '?explod=' + Math.random(4))
```

Comment:
explod is not a word

Reply:
it doesn't really matter because the query string is there in order to make the gif reliably start from the beginning without crapping out other gifs. fixed anyway in f2c83c5